starfish/image/_stack.py

```diff
@@ -22,24 +22,22 @@ def __init__(self, data, tile_format, num_hybs, num_chs, tile_shape):
             self._is_volume = True
 
     @classmethod
-    def from_org_json(cls, org_json_path):
-        with open(org_json_path, 'r') as in_file:
-            org_json = json.load(in_file)
-        metadata_dict = org_json['metadata']
-        num_hybs = metadata_dict['num_hybs']
-        num_chs = metadata_dict['num_chs']
-        tile_shape = tuple(metadata_dict['shape'])
+    def from_image_stack(cls, image_stack_json_path):
+        base_path = os.path.dirname(image_stack_json_path)
 
-        data = numpy.zeros((num_hybs, num_chs) + tile_shape)
-        tile_format = ImageFormat[metadata_dict['format']]
+        with open(image_stack_json_path, 'r') as in_file:
+            image_stack = json.load(in_file)
 
-        data_dicts = org_json['data']
-        base_path = os.path.dirname(org_json_path)
+        num_hybs = len(set(tile['coordinates']['hyb'] for tile in image_stack['tiles']))
+        num_chs = len(set(tile['coordinates']['ch'] for tile in image_stack['tiles']))
+        tile_shape = tuple(image_stack['legend']['default_tile_shape'])
+        tile_format = ImageFormat[image_stack['legend']['default_tile_format']]
+        data = numpy.zeros((num_hybs, num_chs) + tile_shape)
 
-        for data_dict in data_dicts:
-            h = data_dict['hyb']
-            c = data_dict['ch']
-            fname = data_dict['file']
+        for tile_dict in image_stack['tiles']:
```

Comment:
Is there a more descriptive name we could use than `tile_dict`? This contains all the information about the provenance of a tile... 

@dganguli What would you call this? I think this is an object we'll reference a lot in vignettes / notebooks. `tile_metadata`?